analytics/population/nc_pop_recent.ipynb

```diff
@@ -0,0 +1,2597 @@
+{
```

Comment:
<p>Could we group by "GEOID20" instead? Just curious whether they serve the same purpose.</p><p> </p><p>We should also state that we are assuming that people are uniformly distributed within a census tract.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hackforla/311-data/pull/1336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Yes, we can group by GEOID20 too. They serve the same purpose. GEOID20 gives you like a geography id (geographic identifier) - the state, the county followed by the census tract number/code. I grouped them by TRACTCE20 as it directly gives you the census tract number and I have also added this as a tooltip in the map that is being generated.</p><p> </p><p>Added the comment about people being uniformly distributed.  </p>